### PR TITLE
[New] LAMP guides 1k

### PR DESCRIPTION
**Pull request by Rajakavitha1**
- created new CentOS 8
   MariaDB because there is no MySQL package
   added Testing steps 
- created new Debian 10 (Buster)
  MariaDB because there is no MySQL package
 added testing steps
- updated Ubuntu 18.04 
Note to the reviewer: Please validate the suggestion made on DisQus for ubuntu 18.04 guide:MySQL 2 step missed: GRANT ALL <b>PRIVILEGES</b> ON..

- in the Debian guide please check if you need to use 'sudo mysql -u root -p' to login as a root user.

### CODE REVIEW

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit 23832832ecc46aee20afadb7739164e56ac6bd68

https://deploy-preview-3141--nostalgic-ptolemy-b01ab8.netlify.com

**Comment by leslitagordita** [comment]
* @Rajakavitha1 I updated the Centos 8 guide to use the `sites-available` and `sites-enabled` convention used with other distros. 
* There were small changes required in the virtual hosts files and other misc. changes related to CentOS 8 missing an enabled PHP service.
* I updated and tested the CentOS 7 version to include the changes I added to the CentOS 8 version. Some of the other 1k docs link to the CentOS 7 version, so I wanted that to be updated too.
* I also applied our URL structure to the guides included in this PR, i.e. `how-to-install-a-lamp-stack-on-distro-x`

**Comment by Rajakavitha1** [comment]
Thanks @leslitagordita !!! the updates are awesome.

**Comment by Rajakavitha1** [comment]
@leslitagordita  just made a minor update to the location of the .conf file in Debian guide.

File: docs/web-servers/lamp/how-to-install-a-lamp-stack-on-debian-10/index.md [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+Ensure that all values are uncommented, by making sure they do not start with a semicolon (**;**).
+{{< /note >}}
+
+1.  Create the log directory for PHP and give ownership to the Apache user (`www-data`):
+
+        sudo mkdir /var/log/php
+        sudo chown www-data /var/log/php
+
+    {{< note >}}
+If you plan on using your LAMP stack to host a WordPress server, install additional PHP modules: `sudo apt install php-curl php-gd php-mbstring php-xml php-xmlrpc`
```
Comment (andystevensname) [inline docs/web-servers/lamp/how-to-install-a-lamp-stack-on-debian-10/index.md]: I think php-curl is installed on line 237

File: docs/web-servers/lamp/how-to-install-a-lamp-stack-on-debian-10/index.md [hunk 2]
```diff
@@ -0,0 +1,308 @@
+---
+author:
+  name: Linode
+  email: docs@linode.com
+description: 'Get your website or web application online by setting up Apache, MariaDB, and PHP'
```
Comment (andystevensname) [inline docs/web-servers/lamp/how-to-install-a-lamp-stack-on-debian-10/index.md]: Should we add "on Debian 10" and "LAMP stack" to the description to help for SEO?

File: docs/web-servers/lamp/how-to-install-a-lamp-stack-on-centos-7/index.md [hunk 3]
```diff
@@ -2,15 +2,16 @@
...
     name: Joel Kruger
     email: docs@linode.com
-description: 'Create a LAMP stack on a CentOS 7 Linode.'
+description: 'Install a LAMP stack on a CentOS 7 Linode. A LAMP stack includes Linux, Apache, MariaDB, and PHP.'
 keywords: ["LAMP", "CentOS", "CentOS 7", "apache", "mysql", "php", "centos lamp"]
 license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
-modified: 2018-06-21
+modified: 2020-02-20
 modified_by:
     name: Edward Angert
```
[the remark this answers is not in the document]
Comment (leslitagordita) [reply docs/web-servers/lamp/how-to-install-a-lamp-stack-on-centos-7/index.md]: fixed

File: docs/web-servers/lamp/how-to-install-a-lamp-stack-on-debian-10/index.md (hunk 1 above)
Comment (leslitagordita) [reply docs/web-servers/lamp/how-to-install-a-lamp-stack-on-debian-10/index.md]: fixed

File: docs/web-servers/lamp/how-to-install-a-lamp-stack-on-debian-10/index.md (hunk 2 above)
Comment (leslitagordita) [reply docs/web-servers/lamp/how-to-install-a-lamp-stack-on-debian-10/index.md]: Updated description...fixed

**Pull request merged**